disclosure.md

```diff
@@ -0,0 +1,51 @@
+# Responsible Disclosure Policy
+
+18F Responsible Disclosure Policy 
+
+We take the security of our systems seriously, and we value the security community. The responsible disclosure of security vulnerabilities helps us ensure the security and privacy of our users.
+
+**Guidelines**  
+
+We require that all researchers:
+* Make every effort to avoid privacy violations, degradation of user experience, disruption to production systems, and destruction of data during security testing;
+* Perform research only within the scope set out below; 
+* Use the identified communication channels to report vulnerability information to us; and
+* Keep information about any vulnerabilities you’ve discovered confidential between yourself and 18F until we’ve had 90 days to resolve the issue.
+
+
+If you follow these guidelines when reporting an issue to us, we commit to:
+* Not pursue or support any legal action related to your research;
```

Comment:
Per OGC, we cannot commit to not pursuing legal action against anyone. That's DOJ's purview. However, we can authorize access to systems. This is an important distinction.